.golangci.yml

```diff
@@ -109,7 +109,7 @@ linters:
     - errcheck # Errcheck is a program for checking for unchecked errors in go programs. These unchecked errors can be critical bugs in some cases [fast: false, auto-fix: false]
     - errorlint # Errorlint is a linter that can be used to find code that will cause problems with the error wrapping scheme introduced in Go 1.13.
     - exhaustive # check exhaustiveness of enum switch statements [fast: false, auto-fix: false]
-    - exportloopref # checks for pointers to enclosing loop variables [fast: false, auto-fix: false]
+    - copyloopvar # copyloopvar is a linter detects places where loop variables are copied. Replaces exportloopref since Go1.22
```

Comment:
change to "copyloopvar is a linter **that** detects places where ..."

Reply:
Ah, I see this has already been merged. Please hold off on merging next time until the Cloud docs review window has closed, thanks

Reply:
My bad, didn't see that there were docs changes.
I didn't write this text; it's a copy-paste from the [documentation](https://golangci-lint.run/usage/linters/#copyloopvar).